components/raftstore/src/store/fsm/apply.rs

```diff
@@ -1809,6 +1809,7 @@ where
         let (key, value) = (req.get_put().get_key(), req.get_put().get_value());
         // region key range has no data prefix, so we must use origin key to check.
         util::check_key_in_region(key, &self.region)?;
+        let region_id = 1; // Simplify everything.
```

Comment:
```suggestion
        let region_id = self.region.get_id();
```

Reply:
@SpadeA-Tang This is done for testing purposes because we do not initialize all regions. I'll work around this with the test and initialize the regions correctly.